### PR TITLE
Remove time

### PR DESCRIPTION
**Pull request by sjparkinson**
Relates to #454, might be overkill though.

### CODE REVIEW

**Comment by sjparkinson** [comment]
The only place that'd use this information anyway is [next-encryption-api](https://github.com/Financial-Times/next-encryption-api), [and it doesn't seem to](https://github.com/Financial-Times/next-encryption-api/search?utf8=%E2%9C%93&q=timezone).

File: src/lib/check-failing.js [hunk 1]
```diff
@@ -7,8 +7,23 @@ const the = {
 	failures: []
 };
 
-module.exports.init = function () {
+let getTimezone = () => {
```
Comment (bjfletcher) [inline src/lib/check-failing.js]: go for immutability where possible... `const` instead of `let`

File: src/lib/check-failing.js [hunk 2]
```diff
@@ -7,8 +7,23 @@ const the = {
...
+	if (process.env.TZ) {
+		return process.env.TZ;
+	}
+
+	if (fs.existsSync('/etc/timezone')) {
+		return fs.readFileSync('/etc/timezone');
```
Comment (bjfletcher) [inline src/lib/check-failing.js]: you reckon this file can't be blank?

File: src/lib/check-failing.js [hunk 3]
```diff
@@ -7,8 +7,23 @@ const the = {
...
+	}
+
+	if (fs.lstatSync('/etc/localtime').isSymbolicLink()) {
+		return fs.readlinkSync('/etc/localtime').replace('/usr/share/zoneinfo/', '');
```
Comment (bjfletcher) [inline src/lib/check-failing.js]: ditto, worth not returning if it's blank

File: src/lib/check-failing.js [hunk 4]
```diff
@@ -7,8 +7,23 @@ const the = {
...
+	}
+
+	console.error('Error occurred when evaluating the system timezone'); // eslint-disable-line
```
Comment (bjfletcher) [inline src/lib/check-failing.js]: I'd replace with:
`logger.error({ event: 'EXPRESS_GET_TIMEZONE_FAIL' });`
then return `new Date().getTimezone()` (or whatever is the fix in my PR)

**Pull request merged**